mappings/net/minecraft/block/AnvilBlock.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_2199 net/minecraft/block/AnvilBlock
 	FIELD field_17349 TITLE Lnet/minecraft/class_2561;
+	FIELD field_30993 FALLING_BLOCK_ENTITY_DAMAGE_AMOUNT F
+	FIELD field_30994 FALLING_BLOCK_ENTITY_DAMAGE_MAX I
```

Comment:
These are from nbt yes, but we should really make them `maxXxx` `xxxMultiplier` (this amount is per block multiplier)